financialaid/data/country_to_currency.csv

```diff
@@ -0,0 +1,250 @@
+country,currency
+AF,AFN
+AL,ALL
+DZ,DZD
+AS,USD
+AD,EUR
+AO,AOA
+AI,XCD
+AQ,
+AG,XCD
+AR,ARS
+AM,AMD
+AW,AWG
+AU,AUD
+AT,EUR
+AZ,AZN
+BS,BSD
+BH,BHD
+BD,BDT
+BB,BBD
+BY,BYR
+BE,EUR
+BZ,BZD
+BJ,XOF
+BM,BMD
+BT,INR
+BO,BOB
+BA,BAM
+BW,BWP
+BV,
+BR,BRL
+IO,
+VG,USD
+BN,BND
+BG,BGN
+BF,XOF
+BI,BIF
+KH,KHR
+CM,XAF
+CA,CAD
+CV,CVE
+BQ,USD
+KY,KYD
+CF,XAF
+TD,XAF
+CL,CLP
+CN,CNY
+CX,
+CC,
+CO,COP
+KM,KMF
+CG,XAF
+CD,
+CK,NZD
+CR,CRC
+HR,HRK
+CU,CUP
+CW,ANG
+CY,EUR
+CZ,CZK
+CI,XOF
+DK,DKK
+DJ,DJF
+DM,XCD
+DO,DOP
+EC,USD
+EG,EGP
+SV,USD
+GQ,XAF
+ER,ERN
+EE,EUR
+ET,ETB
+FK,FKP
+FO,
+FJ,FJD
+FI,EUR
+FR,EUR
+GF,EUR
+PF,XPF
+TF,
+GA,XAF
+GM,GMD
+GE,GEL
+DE,EUR
+GH,GHS
+GI,GIP
+GR,EUR
+GL,DKK
+GD,XCD
+GP,EUR
+GU,USD
+GT,GTQ
+GG,GBP
+GN,GNF
+GW,XOF
+GY,GYD
+HT,USD
+HM,
+HN,HNL
+HK,
+HU,HUF
+IS,ISK
+IN,INR
+ID,IDR
+IR,IRR
+IQ,IQD
+IE,EUR
+IM,GBP
+IL,ILS
+IT,EUR
+JM,JMD
+JP,JPY
+JE,GBP
+JO,JOD
+KZ,KZT
+KE,KES
+KI,AUD
+KW,KWD
+KG,KGS
+LA,LAK
+LV,EUR
+LB,LBP
+LS,ZAR
+LR,LRD
+LY,LYD
+LI,CHF
+LT,EUR
+LU,EUR
+MO,MOP
+MK,MKD
+MG,MGA
+MW,MWK
+MY,MYR
+MV,MVR
+ML,XOF
+MT,EUR
+MH,USD
+MQ,EUR
+MR,MRO
+MU,MUR
+YT,EUR
+MX,MXN
+FM,USD
+MD,MDL
+MC,EUR
+MN,MNT
+ME,EUR
+MS,XCD
+MA,MAD
+MZ,MZN
+MM,MMK
+NA,ZAR
+NR,AUD
+NP,NPR
+NL,EUR
+NC,XPF
+NZ,NZD
+NI,NIO
+NE,XOF
+NG,NGN
+NU,NZD
+NF,AUD
+KP,KPW
+MP,USD
+NO,NOK
+OM,OMR
+PK,PKR
+PW,USD
+PS,
+PA,USD
+PG,PGK
+PY,PYG
+PE,PEN
+PH,PHP
+PN,NZD
+PL,PLN
+PT,EUR
+PR,USD
+QA,QAR
+RO,RON
+RU,RUB
+RW,RWF
+RE,EUR
+WS,WST
+SM,EUR
+SA,SAR
+SN,XOF
+RS,RSD
+SC,SCR
+SL,SLL
+SG,SGD
+SX,ANG
+SK,EUR
+SI,EUR
+SB,SBD
+SO,SOS
+ZA,ZAR
+GS,
+KR,KRW
+SS,SSP
+ES,EUR
+LK,LKR
+BL,EUR
+SH,SHP
+KN,XCD
+LC,XCD
+MF,EUR
+PM,EUR
+VC,XCD
+SD,SDG
+SR,SRD
+SJ,NOK
+SZ,SZL
+SE,SEK
+CH,CHF
+SY,SYP
+ST,STD
+TW,
+TJ,TJS
+TZ,TZS
+TH,THB
+TL,USD
+TG,XOF
+TK,NZD
+TO,TOP
+TT,TTD
+TN,TND
+TR,TRY
+TM,TMT
+TC,USD
+TV,AUD
+UM,
+VI,USD
+GB,GBP
+US,USD
+UG,UGX
+UA,UAH
+AE,AED
+UY,UYU
+UZ,UZS
+VU,VUV
+VA,EUR
+VE,VEF
+VN,VND
+WF,XPF
+EH,MAD
+YE,YER
+ZM,ZMW
+ZW,ZWL
+AX,EUR
```

Comment:
We'll do it later when it's used, given that Peter said this is lower priority now